spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkWrite.java

```diff
@@ -213,6 +214,7 @@ private void abort(WriterCommitMessage[] messages) {
     if (cleanupOnAbort) {
       Map<String, String> props = table.properties();
       Tasks.foreach(files(messages))
+          .executeWith(ThreadPools.getWorkerPool())
```

Comment:
@RussellSpitzer, what do you think about this change?

Reply:
Seems fine to me, although I really wish we had a more discrete pool for things like this. I find that our worker pool is hard to configure for end users.

Reply:
Should we add a Spark-managed pool? We have the ability to plug in the pool used in most places now, thanks to @yittg.